test/java/com/rapleaf/jack/test_project/database_1/models/Comment.java

```diff
@@ -33,7 +33,7 @@
 
 public class Comment extends ModelWithId<Comment, IDatabases> implements Comparable<Comment>{
   
-  public static final long serialVersionUID = 8036617193786054443L;
+  public static final long serialVersionUID = 6213989608937906012L;
```

Comment:
I'm guessing this change is due to this being generated with ruby 1.9.3. Currently this project is still expecting ree 1.8.7. Can you generate with that and see if that keeps the ids the same?